tests/integration/dab_rbac/test_read_permissions.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+from django.contrib.contenttypes.models import ContentType
+
+from rest_framework.reverse import reverse
+
+from ansible_base.rbac.models import RoleDefinition
+
+from aap_eda.core import models
+
+
+@pytest.fixture
+def activation(user):
+    return models.Activation.objects.create(name="activation", user=user)
+
+
+@pytest.fixture
+def rulebook_process(activation):
+    return models.RulebookProcess.objects.create(name="test-instance", activation=activation)
+
+
+@pytest.mark.django_db
+def test_get_activation(activation, user, user_api_client):
+    rd = RoleDefinition.objects.create_from_permissions(
```

Comment:
@AlanCoding I think in the current EDA implementation we are not using the Django permission model there is an internal [implementation](https://github.com/ansible/eda-server/blob/7e956b8810987fce20de698ad05a7f3e3acc323f/src/aap_eda/core/models/auth.py#L44) and post migration there is setup [here](https://github.com/ansible/eda-server/blob/7e956b8810987fce20de698ad05a7f3e3acc323f/src/aap_eda/core/management/commands/create_initial_data.py#L26). Is the DAB role/permission implementation along the same lines or is it more in line with the Django Permission model.

Reply:
Yes, I considered using the exiting permission model but it did not look good. Both the existing permission and role models are unused as of this patch. They could ultimately be deleted after all open questions are resolved. The new permission model has the exact same schema as the Django permission model. It would be possible to use the Django permission model but reasons not to are (1) it includes permissions for models not managed by RBAC and (2) a custom model is easier to modify later.